### DISCUSSION TITLE
useTransition: TypeError: Cannot read property 'ref' of null

### DISCUSSION THREAD

**Discussion by leo-petrucci** [pmndrs/react-spring#990] [Support] [answered]
I'm on the latest `react-spring@^9.0.0-rc.2`. When I try to use useTransition I get the following error.

![Screenshot from 2020-05-07 11-12-09](https://user-images.githubusercontent.com/23533178/81283258-3fc8d380-9054-11ea-96e7-544bd3a8ab8e.png)

This code has worked before and it does work with 8.x, unfortunately using 8.x breaks other things in my script, so I can't go back to that. 

I have no idea what's wrong with it, I haven't even updated React Spring, but it all of a sudden broke and refuses to work. The code is exactly the same as the useTransition example:

```
  const transitions = useTransition(open, null, {
    from: { transform: `scale(1.5)`, opacity: 0 },
    enter: { transform: `scale(1)`, opacity: 1 },
    leave: { transform: `scale(1.5)`, opacity: 0 },
    unique: true
  });

  return (
    <>
      <div
      >
        {transitions.map(({ item, key, props }) =>
          item &&
            <>
              <animated.div
                style={props}
              >
              </animated.div>
            </>
        )}
      </div>
    </>
  );
```

What's wrong with it?

**Comment by leo-petrucci** [comment]
Codesandbox with example
https://codesandbox.io/s/mystifying-kowalevski-zyiqm?file=/src/index.js

**Comment by aleclarson** [comment, marked as answer]
You're using the old `useTransition` API. See here: https://aleclarson.github.io/react-spring/v9/breaking-changes/
